speech/unit_tests/test__gax.py

```diff
@@ -36,13 +36,8 @@ def _call_fut(self, sample, language_code, max_alternatives,
     def test_ctor(self):
         from google.cloud import speech
         from google.cloud.speech.sample import Sample
-        from google.cloud.grpc.speech.v1beta1.cloud_speech_pb2 import (
-            SpeechContext)
-        from google.cloud.grpc.speech.v1beta1.cloud_speech_pb2 import (
-            RecognitionConfig)
-        from google.cloud.grpc.speech.v1beta1.cloud_speech_pb2 import (
-            StreamingRecognitionConfig)
-        from google.cloud.grpc.speech.v1beta1.cloud_speech_pb2 import (
+        from google.cloud.proto.speech.v1beta1.cloud_speech_pb2 import (
+            RecognitionConfig, SpeechContext, StreamingRecognitionConfig,
```

Comment:
I checked the internal style guide, and it does allow this (but prefers importing `cloud_speech_pb2` and using everything off of that, which I am not going to change because of the unwieldily-long names).